pkg/webhook/resources/templateversion/validator.go

```diff
@@ -75,6 +78,19 @@ func (v *templateVersionValidator) Create(_ *types.Request, newObj runtime.Objec
 		}
 	}
 
+	// Check JSON data in the annotations. This must be valid JSON data, as
+	// otherwise the IndexFunc of the cache couldn't process the VMTemplateVersion
+	volumeClaimTemplateString, ok := vmTemplVersion.Spec.VM.ObjectMeta.Annotations[util.AnnotationVolumeClaimTemplates]
+	if ok && volumeClaimTemplateString != "" {
```

Comment:
Could you add some test codes to cover those cases? thanks.

Reply:
Done